llama.cpp

```diff
@@ -8123,20 +8156,26 @@ struct llm_build_context {
                 // skip computing output for unused tokens
                 struct ggml_tensor * inp_out_ids = build_inp_out_ids();
                 cur   = ggml_get_rows(ctx0,   cur, inp_out_ids);
+                inpL  = ggml_get_rows(ctx0,  inpL, inp_out_ids);
                 inpSA = ggml_get_rows(ctx0, inpSA, inp_out_ids);
             }
-
-            struct ggml_tensor * ffn_inp = ggml_add(ctx0, cur, inpSA);
-            cb(ffn_inp, "ffn_inp", il);
+            struct ggml_tensor * attn_out = cur;
+            // only used for non-parallel residual
```

Comment:
This comment is no longer relevant as of https://github.com/ggerganov/llama.cpp/pull/6635/commits/8a15f932af45f9532bbec48dac7171677cf5616a

```suggestion
```

Reply:
Removed thanks !